src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -405,7 +405,11 @@
 
 @private commaSeparatedFields(args)
     @join arg : args on ", "
-        {@variableField(arg)}
+        @if or(arg.type.isMessage, arg.type.isEnum)
+            print_r({@variableField(arg)}), true)
```

Comment:
I think you may have an extra right paren here....

Reply:
Yes! Good catch